go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/containerd/nerdctl
 
-go 1.19
+go 1.20
```

Comment:
@AkihiroSuda, curious on your thoughts if we should upgrade the min Go version or give folks a little more time to migrate to supported Go versions. i.e. Go 1.20 and 1.21 now

Reply:
We still have to support 1.20